src/test/java/uk/gov/justice/hmpps/prison/service/BookingServiceTest.java

```diff
@@ -231,7 +230,7 @@ public void verifyCanViewSensitiveBookingInfo() {
 
     @Test
     public void verifyCanViewSensitiveBookingInfo_systemUser() {
-        when(authenticationFacade.isOverrideRole(any())).thenReturn(true);
+        when(authenticationFacade.isOverrideRole(any(String[].class))).thenReturn(true);
```

Comment:
Mocks for vararg parameters [changed in Mockito 5](https://github.com/mockito/mockito/releases/tag/v5.0.0) so that you have to either match the correct number of args or instead use an array type.